tests/budget_accounting_test.py

```diff
@@ -467,7 +471,26 @@ class ComputeBudgetTestCase:
                         weight=8,
                         sensitivity=3),
                 ],
-                expected_pipeline_noise_std=40.02)
```

Comment:
should we write/point to the formulas in the class definition?

Reply:
Yeah, we should write documentation on PLD accountant. In this case, there is no simple formula anyway :)